controllers/graphController.js

```diff
@@ -141,6 +140,12 @@ GraphController.prototype.embed = function(req, res, next) {
 	}).catch(next)
 }
 
+// "moon-test-one" -> "Moon Test One"
+function convNameToHumanReadable(name) {
```

Comment:
It also uppercases every word, so it's kinda specific to the usecase, hence why I named it this way, if we want to change the functionality later, then we would need to change the function name also, don't want to tie too much specifics into any function name.